src/OpenTK/Platform/Tizen/ITizenWindowAttributes.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenTK.Platform.Tizen
+{
+    /// <summary>
+    /// Describes Tizen window attributes.
+    /// </summary>
+    public interface ITizenWindowAttributes
+    {
+        /// <summary>
+        /// Get or set whether the window accepts a focus or not.
+        /// </summary>
+        bool IsFocusAllowed { get; set; }
+
+        /// <summary>
+        /// Set opacity of the window
+        /// </summary>
+        float WindowOpacity { set; }
```

Comment:
The getter of WindowOpacity is needed also. 
You can use below SDL method.

/**
 *  \brief Get the opacity of a window.
 *
 *  If transparency isn't supported on this platform, opacity will be reported
 *  as 1.0f without error.
 *
 *  \param window The window in question.
 *  \param out_opacity Opacity (0.0f - transparent, 1.0f - opaque)
 *
 *  \return 0 on success, or -1 on error (invalid window, etc).
 *
 *  \sa SDL_SetWindowOpacity()
 */
extern DECLSPEC int SDLCALL SDL_GetWindowOpacity(SDL_Window * window, float * out_opacity);